codes/utils.py

```diff
@@ -17,6 +17,8 @@
 adm3_dir_fmt = 'gadm36_{iso3}_{datestamp}.zip'
 
 CUM_CASE_MIN_FILTER = 10
+PROCESSED_DATA_ERROR_HANDLING = "warn"
```

Comment:
let's actually change the default to "raise" and then if it starts breaking people's code we can decide if either the datasets need to be changed or we need to relax this error checking.

Reply:
Sounds good!